Added Contributing.md

PukieDiederik opened the pull request:
## What has been implemented:
- Added a CONTRIBUTING.md file

> Added a CONTRIBUTING.md file

This file should be used as a reference and guide to creating and reviewing pull requests.

## Related issues:
- #12

gleal42 commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,92 @@
+- Brief description of what has been implemented (2)
+- Brief description of what has been implemented (3)
+
+> Brief description of what has been implemented (1)
+
+Bigger explanation of what has changed. This can also include reasoning why it has changed.
```
Suggestion: To avoid repeating the commit messages
```suggestion
> (1) Bigger explanation of what has changed. This can also include reasoning why it has changed.
```

gleal42 commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,92 @@
+
+---
+
+Make sure to add labels and requests reviews once your pull request is created.
```
Maybe explain the logic behind each label here?

gleal42 reviewed (approved):
## Issue(s) found:
N/A

## Style related issue(s) found:
- Commit messages duplicated for bigger explanation
- Using `>` is usually for footnotes and things of lower importance

## Notes:
Looks great! Changes are optional!

PukieDiederik commented:
## What has been implemented:
1. Commit messages are not being duplicated anymore
2. Changed bullet point list to numbered list for "What has been implemented".
3. Stopped using `>`
4. Expanded documentation for labels.


1/2/3 - I changed it to a numbered list so we can reference each change by number. I use `n - ` for the notes because other things would turn it into a numbered list which does not allow you to skip numbers.

Dmarceli commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,85 @@
+
+---
+
+Make sure to add labels and requests reviews once your pull request is created. Labels can be added on the right when creating a pull request (or after having created one). This is also where you can added reviewers and relate issues. Labels are usually similar to the labels from related issues.
```
have we used labels so far?

Dmarceli commented:
seems good

gleal42 reviewed (approved):
LGTM!

ncameiri reviewed (approved):
Good approach to give PR more info
